participant-manager/src/app/site-coordinator/location/edit-location/edit-location.component.scss

```diff
@@ -0,0 +1,3 @@
+.modal__body .form-control[readonly] {
```

Comment:
Should this have copyright header?

Reply:
Hi @jkrogers18,  we include copyright headers in only SVG images, not in HTML and CSS files.